meeting-notes/2020/Q2/2020-04-08--gui-and-browsers-biweekly.md

```diff
@@ -0,0 +1,53 @@
+ # GUI and In Web Browsers 2020-03-25
+
+A biweekly discussion and show & tell around IPFS GUIs and Web Browsers
+
+* Details: https://github.com/ipfs/team-mgmt/issues/790
+
+## Participants
+
+- @autonome
+- @jacobheun
+- @jessicaschilling 
+- @vasco-santos
+- @lidel
+
+## Recording
+
+- https://youtu.be/b4BBbpl-BnQ 
+- https://ipfs.io/ipfs/bafybeidop5y6yu7j64vnzopi3xqij7nz3ptdqszyxi6z3n4j5si5bldqzq/
+
+## Agenda
+
+- Quick update about ipfs-companion [v2.11.0](https://github.com/ipfs-shipyard/ipfs-companion/releases/tag/v2.11.0) (details in release ntoes)
+- go-ipfs 0.5 release preparation: 
+    - https://github.com/ipfs/go-ipfs/issues/7109
+    - POST-only HTTP API in go-ipfs 0.5
+        - If you use custom HTTP Client, make sure it sends POST requests
+        - JS land: https://github.com/ipfs/js-ipfs/issues/2971
+- proposal: more breathing room, switch this call to bi-weekly?
+    - I heard rumors about no meetings on Wednesdays, do we need to pay attention to this?
+    - We will switch to bi-weekly and move the meeting day if that becomes needed.
+- Cliqz is interested in IPFS integration, talking soon
+- Web-RTC, ideas for grants?
+    - Flushing out [distributed signaling spec](https://github.com/libp2p/specs/pull/159)
+        - Create a POC of distributed signaling
+    - Distributed ICE
+    - Determine reasonable connection manager constraints for webrtc
+    - An "audit" of the things we have planned for webrtc, ie: What is reasonable and what are features/experiments we should have on our roadmap.
+    - using https://github.com/ipfs-shipyard/ipfs-share-files/issues/58 as a playground?
+- dweb.link traffic around Opera launch - https://a92e77763d034e46b95d526b2d192678.us-east-1.aws.found.io:9243/app/kibana#/visualize/edit/cc78b050-72b5-11ea-a22c-fd6c76652285?_g=(filters%3A!()%2CrefreshInterval%3A(pause%3A!t%2Cvalue%3A0)%2Ctime%3A(from%3Anow-2w%2Cto%3Anow))
```

Comment:
@autonome is this a public link? When I open it in incognito asks me for Google Account.
Should we remove it?